pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   collection: ^1.15.0
   flutter:
     sdk: flutter
+  http: ^0.13.4
```

Comment:
The way I implemented it - they don't, since the provisioning is done in base package. But I've suggested different approaches in https://github.com/ably/ably-flutter/pull/343#issuecomment-1060899274